development/image-syncer/README.md

```diff
@@ -0,0 +1,62 @@
+# image-syncer
+
+## Overview
+
+ image-syncer is used to **safely** copy container images from one registry to another. 
+The main use case is to preserve images from third party registries in our own registry that we can rely on.
+
+Syncing process steps:
+1. Pull image from source.
+2. Check if image exists in target.
+3a. If image does not exist: re-tag image and push to target registry.
+3b. If image exists: compare IDs. If they are different, synchronization is interrupted. It means that something is wrong and the source image was changed (this should not happen).
+
+These steps guarantee that images in our registry are immutable.
+
+## Usage
+
+To run it, use:
```

Comment:
```suggestion
To run image-syncer, use:
```
Can we replace `it` with `The image-syncer package` or `image-syncer` (whichever you think is better) to make the sentence clearer and more descriptive?